lib/bokken/periodically.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Bokken.Periodically do
+  @moduledoc """
+  Executes a task periodically.
+  """
+  use GenServer
```

Comment:
We could replace our own job scheduler written in this `GenServer` with [`:quantum`](https://hexdocs.pm/quantum/readme.html). It is really simple and could leave us the ready for future periodical tasks.

For more complicated and demanding cronjobs we could use [:oban](https://hexdocs.pm/oban/Oban.html), but I don't think it is need in this case.

Reply:
I'm not being able to config the suggestion... what am I doing mostly wrong? :sweat_smile: